cmake/LYTestWrappers.cmake

```diff
@@ -261,9 +261,19 @@ function(ly_add_test)
     # Check to see whether or not this test target has been stored in the global list for walking by the test impact analysis framework
     get_property(all_tests GLOBAL PROPERTY LY_ALL_TESTS)
     if(NOT "${test_target}" IN_LIST all_tests)
+        # Extract the test target name from the namespace::target_name composite
+        string(REPLACE "::" ";" test_components ${test_target})
+        list(LENGTH test_components num_test_components)
+        if(num_test_components GREATER 1)
+            list(GET test_components 1 test_name)
+        else()
+            set(test_name ${test_components})
+        endif()
+        # Store the test target name sans namespace so they can be looked up without the preceeding namespace
+        set_property(GLOBAL APPEND PROPERTY O3DE_ALL_TESTS_DE_NAMSPACED ${test_name})
```

Comment:
update: `O3DE_ALL_TESTS_DE_NAMSPACED` -> `O3DE_ALL_TESTS_DE_NAMESPACE` (missing E)

Reply:
Good to know for future uses, although in this specific RapidJSON instance it's now redundant as per @burelc-amzn 's suggestion 👍